dist/README.md

```diff
@@ -182,41 +182,39 @@ For further instructions on these variables as well as configuring
 authentication, consult [our documentation for
 self-hosting](https://support.getgrist.com/self-managed/).
 
-# updating Grist
+# Upgrading Grist
 
-To update to the latest Grist version at any time, first stop Grist.
-
-If using systemd:
+To upgrade to the latest Grist version, we recommend using systemd as
+described above, and enabling the supplied systemd timer:
 
 ```sh
-sudo systemctl disable --now grist
+sudo systemctl enable --now grist-upgrade.timer
 ```
 
-Otherwise:
+This timer will check and apply Grist upgrades every weekend around
+the server time's midnight.
```

Comment:
Maybe:
```suggestion
This timer will restart Grist and apply Grist upgrades every Saturday night at around
the server time's midnight.
```
Key changes:
- Mentioning it's Saturday night specifically
- Mentioning the service will restart / otherwise convey it's unavailable for a bit